docs/learn/lagandpacketloss.md

```diff
@@ -1,109 +1,56 @@
 
-    `Propagation Delay = 0.3-ns * meters-to-travel`
+A ping is a simplified way of measuring RTT that involves sending a very basic message, with no processing at either end of the interaction, to get a general idea of network responsiveness and latency.
 
-    In practice that means that even under ideal conditions, it would take at least 50ms for a packet to go from Montreal to Australia, and approximately 15ms to cross the USA east to west.
-
-    This type of delay is optimizable in a client-server game - by placing servers closer to your players you effectively reduce propagation delay. In case dispersing servers isn't an option for gameplay reasons (for instance when your players NEED to be able to play with as little latency across, say, the entire US) an even more expensive and performant option is possible - a so-called edge network can be built.
-
- :::funfact
-    Riot games famously had to go that route for their League of Legends, building their own network that peered with ISPs across North America to provide the fastest connection possible and reducing network hops as much as possible.
- :::
-
-In the context of networking it's valuable to consider the combination of network latency factors which is typically referred to as Ping or Round Trip Time (RTT).
-
-## Round Trip Time (RTT)
-RTT is the time it takes for a packet to travel from one host to another and then for a response packet to travel back. This includes both the two-way sum of network latency factors and, as it contributes to how quickly the server can send out a response, the frame rate of the remote host,.
-
-Since the traffic is unlikely to travel at the same speed in each direction, the RTT is rarely exactly the time it takes for a packet to go from one host to another. Regardless, it's a common practice to approximate one-way travel time by dividing RTT by 2.
-
-Another complication arises from the fact that for any two hosts, the RTT time between them isn't a constant. It varies over time, normally hovering around a certain average value. The components of network latency can also vary over time, causing the RTT to deviate from the expected value. This deviation is called Jitter.
-
-## Jitter
+<ImageSwitcher
+lightImageSrc="/ping-animation-light.gif?text=LightMode"
+darkImageSrc="/ping-animation-dark.gif?text=DarkMode"/>
 
-Jitter is the rate at which ping changes over a period of time.  It can affect RTT mitigation and also make packets arrive out of order if it causes  the router to send packets through different routes, which can cause an older packet to arrive before a newer packet.
+The time between sending the request and receiving the answer is the value of your ping. For example, a ping of 20ms means it takes roughly 10ms for data to travel from the client to the server, and then 10ms to travel back. Higher ping values indicate a lot of network latency, which can make games feel slow and unresponsive.
```

Comment:
is the send / recieve time always the same? That's implied here, not sure if it's true though.

Reply:
```suggestion
The time between sending the request and receiving the answer is the value of your ping. Sending and receiving data can take different amounts of time: for example, with a 20ms ping it might take 15ms to send a request from the client to the server and only 5ms to receive a response. Higher ping values indicate a lot of network latency, which can make games feel slow and unresponsive.
```
Better?

Reply:
better, thank :)